doc/influxdb_v2_integration.md

```diff
@@ -0,0 +1,88 @@
+# InfluxDB v2 Integration
+
+The current plugin version uses InfluxDB v1.x authentication semantics (user name and password, and a database), while v2 is using API token, bucket, and organization to regulate the access to the data. 
+
+The current Jenkins InfluxDB Target configuration requires the following: 
+- A Jenkins Credential with username and password, instead of a Secret Text that would better store the ImfluxDB v2 token. 
+- InfluxDB v2 settings: bucket, org, and a secret. If the org is not specified the plugin creates an InfluxDB v1 client. 
+- Performs HTTP Basic authentication with the provided username and password credentials. The password is significant but the username is not which the Basic authentication scheme used by InfluxDB v2 client, which results in an authentication failure. 
+
+The solution is to use InfluxDB Telegraf http listener that serves as a proxy between the influxdb_plugin and InfluxDB v2 instance. The listener can authenticate the influxdb_plugin client using only the password with a Basic schema, and stream the metrics to InfluxDB v2 with the help of influxdb_v2 output plugin, that in turn could perform filtering on the measurements and tags. 
```

Comment:
Since #125 fixed InfluxDB 2.x authentication with username+password, this is not entirely true anymore. You _can_ authenticate to InfluxDB 2.x with username+password even though the standard way of authenticating is by token (which is still under work).

Reply:
Agreed.  I've just tested successfully  username+password authentication.

Initially, this was my stumbling block that was quickly eclipsed by SQ integration issues. Now that i am more familiar with how the plugin works, it is easier to better articulate the requirements for both integrations.